README.md

```diff
@@ -152,27 +181,32 @@ For example, to run the tests for the project `api`, run
 
 To run a single test, use the `testFile` flag as follows:
 
-> > nx run <project-name>:test --testFile <name-of-test-file>
+> > nx text <project-name> --test-file=<name-of-test-file>
 
 e.g.
 
-> > nx run api:test --testFile entities.e2e.spec.ts
+> > nx run api:test --test-file=entities.e2e.spec.ts
 
 To run all tests whose names match a RegEx pattern, run
 
-> > nx run <project-name>:test --testPathPattern <regex-to-match-file-paths>
+> > nx test api --test-path-pattern=<regex-to-match-file-paths>
 
 e.g.
 
-> > nx run api:test --testPathPattern 'src/.\*validate.\*'
+> > nx test api --test-path-pattern='src/.\*validate.\*'
+
+In the event that a Jest snapshot fails, first run the single test with a failing snapshot in isolation. To accept the snapshot changes, rerun the test with the -u flag as follows:
+
+> > nx text <project-name> -- --test-file=<name-of-test-file> -u
```

Comment:
I run it without the extra `--`.  Is it necessary?  Also, `nx test`

Reply:
Yes- you can't pass the -u without the extra `--`.